sevntu-checks/sevntu-checks.xml

```diff
@@ -216,6 +216,14 @@
       <property name="ignoreMethod" value="false"/>
     </module>
 
+    <!-- Jsr305 -->
+    <module name="com.github.sevntu.checkstyle.checks.coding.Jsr305AnnotationsCheck">
+      <property name="packages" value="your.package.path.to.check,another.package.path.to.check"/>
+      <property name="excludePackages" value="your.package.path.not.to.check,another.package.path.not.to.check"/>
```

Comment:
Please make these real values as if you were adding this to sevntu.
Just put `severity` to `ignore` for this check.

Reply:
Done.